contracts/solidity/signature-example/ReceiverPays.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.7.0 <0.9.0;
+// This will report a warning due to deprecated selfdestruct
```

Comment:
Q: should this comment move to the shutdown method?

Reply:
✅  Done